sdl2.vapi

```diff
@@ -2516,20 +2741,30 @@ namespace SDL {
 			[CCode (cname = "SDL_RecordGesture")]
 			public static int record_gesture (Touch.TouchID touch_id);
 
-			[CCode (cname = "SDL_LoadDollarTemplate")]
-			public static int load_dollar_template_rw (Touch.TouchID touch_id, SDL.RWops src);
+			[Version (since = "2.0.0")]
+			[CCode (cname = "SDL_LoadDollarTemplates")]
+			public static int load_dollar_templates_rw (Touch.TouchID touch_id, SDL.RWops src);
 
-			public static int load_dollar_template (Touch.TouchID touch_id, string file) {
-				return load_dollar_template_rw (touch_id, new SDL.RWops.from_file (file, "rb"));
+			[Version (since = "2.0.0")]
+			public static int load_dollar_templates (Touch.TouchID touch_id, string file) {
+				return load_dollar_templates_rw (touch_id, new SDL.RWops.from_file (file, "rb"));
 			}
 
+			[Version (since = "2.0.0")]
 			[CCode (cname = "SDL_SaveDollarTemplate")]
-			public static bool save_dollar_template_rw (GestureID gesture_id, SDL.RWops dst);
+			public static int save_dollar_template_rw (GestureID gesture_id, SDL.RWops dst);
```

Comment:
I think types should match that of the original API as close as possible. I'm thinking that if the SDL developers meant for the return to be strictly boolean, they would have used SDL_bool instead of intto convey the idea. The return value might change in the future to return negative error codes instead (like much of the API already does).